string                  name
geometry_msgs/Pose      pose
int32                   state

int32 STATE_ATTACHED = 1
int32 STATE_FREE = 2

